hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -82,6 +82,8 @@ def __init__(
         unit: str = Units.feet.value,
         country: str = None,
```

Comment:
the country attribute does not seem to be used now

Reply:
this is still not used. Should we delete it?

Reply:
this is the init method that country is not used. It is an argument on the respective methods. So either we need to delete it from here, or save it in self. here and not have it as an argument in the other methods.